packages/base/src/util/dragAndDrop/handleDrop.ts

```diff
@@ -0,0 +1,27 @@
+import type UI5Element from "../../UI5Element.js";
+import type MovePlacement from "../../types/MovePlacement.js";
+import DragRegistry from "./DragRegistry.js";
+
+function handleDrop(e: DragEvent, component: UI5Element, target: HTMLElement, placement: `${MovePlacement}`): void {
+	e.preventDefault();
+	const draggedElement = DragRegistry.getDraggedElement();
+
+	if (!draggedElement) {
+		return;
```

Comment:
same here i guess e.g. dropping files are not allowed, maybe ok, consumer can handled this via attaching native drop events but would be nice to be able use the drop indicator of d&d